packages/treasury/src/stablecoinMachine.js

```diff
@@ -77,13 +76,19 @@ export async function start(zcf) {
   // away fees so the tests show that the funds have been removed.
   const { zcfSeat: rewardPoolSeat } = zcf.makeEmptySeatKit();
 
-  // We provide an easy way for the vaultManager and vaults to add rewards to
-  // this seat, without directly exposing the seat to them.
-  function stageReward(amount) {
-    const priorReward = rewardPoolSeat.getAmountAllocated('RUN', runBrand);
-    return rewardPoolSeat.stage({
-      RUN: AmountMath.add(priorReward, amount, runBrand),
-    });
+  /**
+   * We provide an easy way for the vaultManager and vaults to add rewards to
+   * the rewardPoolSeat, without directly exposing the rewardPoolSeat to them.
+   *
+   * @type {TransferReward}
+   */
+  function transferReward(amount, fromSeat) {
```

Comment:
Ah good point. I was trying not to expose the `rewardPoolSeat`, as in the original design, but you're right that it would be better to have only one `reallocate` call. Let me see if I can accomplish both.

Reply:
Done! Let me know if that looks ok.